distributed/deploy/spec.py

```diff
@@ -11,6 +11,53 @@
 from ..security import Security
 
 
+class SpecProcess:
+    """ An interface for Scheduler and Worker processes for use in SpecCluster
+
+    Parameters
+    ----------
+    loop:
+        A pointer to the running loop.
+
+    """
+
+    def __init__(self, loop=None):
+        self.address = None
+        self.loop = loop
```

Comment:
Is a pointer to the event loop required by SpecCluster in order to run?

Reply:
Yes. I fixed a bug on Monday in ECSCluster where the adaptive code was trying to schedule a callback on the loop.

Reply:
But perhaps the ECSCluster object can have a pointer to its own event loop, rather than relying on Worker/Scheduler objects to have them.

Reply:
Sure. That will require a change in adaptive where is it trying to access the loop of the scheduler.

Reply:
I can't actually seem to find it now.